spec/reserved_words_spec.rb

```diff
@@ -18,17 +18,17 @@ class Nasty#{nasties.length} < ActiveRecord::Base
       nasty = Object.const_get("Nasty#{nasties.length}")
       nasty.class_eval do
         self.primary_key = 'fake_primary_key'
-        col :fake_primary_key
+        col :fake_primary_key, limit: 191
```

Comment:
http://blog.arkency.com/2015/05/how-to-store-emoji-in-a-rails-app-with-a-mysql-database/ for more context.  Basically mysql tracks limit on a byte level even if the character set is multi-byte.  `utf8mb4` requires four bytes per character and "The maximum index length of InnoDB storage engine is 767 bytes", so `191 * 4 = 764`